mmdet/models/detectors/base.py

```diff
@@ -197,7 +199,7 @@ def show_result(self,
         ]
         labels = np.concatenate(labels)
         # draw segmentation masks
-        if segm_result is not None:
+        if segm_result is not None and len(labels) > 0:  # non empty
```

Comment:
What's the case of `segm_result is not None and len(labels) == 0`?

Reply:
When only evaluate bbox, segm_results is None
When prediction is empty, len(labels) = 0, segm_result is not None